### PR TITLE
feat: 🎸 profile linked

### PR DESCRIPTION
**Pull request by alok722**
this commit includes changes related to the team page and associated profiles of individual

### CODE REVIEW

File: src/pages/Team.vue [hunk 1]
```diff
@@ -0,0 +1,346 @@
...
+            </div>
+
+            <h2 class="mt-4 font-bold text-xl underline">you ?</h2>
+
+            <p class="text-xs text-gray-500 text-center mt-3 ellipsis">
+              quote that defines you?
+            </p>
+          </div>
+        </div>
+        <div
```
Comment (MiKr13) [inline src/pages/Team.vue:60]: We should make this a common component as code is the same for members & founders. It just differs at what kind of data we're sending as props.

File: src/pages/Team.vue [hunk 2]
```diff
@@ -0,0 +1,346 @@
...
+    }
+  },
+  computed: {
+    memberData() {
+      if (this.$page && this.$page.allTeam && this.$page.allTeam.edges) {
+        return this.$page.allTeam.edges.filter(
+          (elem) => elem.node.designation !== "founder"
+        )
+      } else {
+        return {}
```
Comment (MiKr13) [inline src/pages/Team.vue:280]: Should be `[]` no?

File: src/pages/Team.vue [hunk 3]
```diff
@@ -0,0 +1,346 @@
...
+      }
+    },
+    founderData() {
+      if (this.$page && this.$page.allTeam && this.$page.allTeam.edges) {
+        return this.$page.allTeam.edges.filter(
+          (elem) => elem.node.designation === "founder"
+        )
+      } else {
+        return {}
```
Comment (MiKr13) [inline src/pages/Team.vue:289]: Should be `[]` no?

File: src/pages/me/[username].vue [hunk 4]
```diff
@@ -1,10 +1,26 @@
 <template>
-  <Layout page="me"> </Layout>
+  <Layout page="me">
+    <div class="container relative mx-auto">
+      <div class="items-center flex flex-wrap">
+        <div class="w-full lg:w-6/12 px-4 ml-auto mr-auto text-center">
+          <div class="py-4">
+            <h1 class="font-semibold text-5xl underline">the resum∈</h1>
```
Comment (MiKr13) [inline src/pages/me/[username].vue:7]: [Optional] Will this: `Résumé` look better & classier?

**Review by MiKr13 — CHANGES_REQUESTED** [review]
Few changes needed but other than that 😍😍💖

File: src/pages/Team.vue (hunk 1 above)
Comment (alok722) [reply src/pages/Team.vue]: PR updated as per your feedback.
Please review & resolve this conversation.

File: src/pages/Team.vue (hunk 2 above)
Comment (alok722) [reply src/pages/Team.vue]: PR updated as per your feedback.
Please review & resolve this conversation.

File: src/pages/Team.vue (hunk 3 above)
Comment (alok722) [reply src/pages/Team.vue]: PR updated as per your feedback.
Please review & resolve this conversation.

File: src/pages/me/[username].vue (hunk 4 above)
Comment (alok722) [reply src/pages/me/[username].vue]: PR updated as per your feedback.
Please review & resolve this conversation.

File: src/pages/Team.vue [hunk 5]
```diff
@@ -1,251 +1,12 @@
...
-    <div ref="joinus" class="container relative mx-auto">
-      <div class="items-center flex flex-wrap">
-        <div class="lg:w-6/12 px-4 ml-auto mr-auto text-center">
-          <div class="py-4">
-            <h1 class="font-semibold text-5xl underline">wanna join?</h1>
-          </div>
-        </div>
-      </div>
-    </div>
+    <PageHeader title="the craftsmen" />
```
Comment (MiKr13) [inline src/pages/Team.vue:3]: Noice ❤

**Review by MiKr13 — APPROVED** [review]
LGTM 💥🎋

**Pull request merged**